neptune/alpha/internal/backends/operations_preprocessor.py

```diff
@@ -140,7 +158,7 @@ def visit_delete_attribute(self, op: DeleteAttribute) -> None:
             else:
                 # This case is tricky. There was no delete operation, but some modifications was performed.
                 # We do not know if this attribute exists on server side and we do not want a delete op to fail.
-                # So we need to send a single modification before delete so be sure a delete op is valid.
+                # So we need to send a single modification before delete to be sure a delete op is valid.
```

Comment:
Or we could make the delete operation idempotent and define its meaning in terms of the end result (after processing a delete operation, the attribute does not exist, whether it existed previously or not) rather than "delete operation deletes an existing attribute"